app/views/posts/index.html.erb

```diff
@@ -8,82 +8,89 @@
-                    <% if !like.nil? %>
-                    <form action="<%= like_path(like)%>" method="post">
-                        <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
-                        <input type="hidden" name="_method" value="DELETE"/>
-                        <input name="like[post_id]" type="hidden" value="<%= post.id %>"/>
-                        <button type="submit">UnLike</button>
-                      </form>
-                    <% else %>
-                      <form action="/likes" method="post">
-                        <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
-                        <input name="like[post_id]" type="hidden" value="<%= post.id %>"/>
-                        <button type="submit">Like</button>
-                      </form>
-                    <% end %> 
-            <% end %>
-          </div>
-          <div class = "button">
-            <button type="button" class="btn btn-default navbar-btn"><%= link_to 'Show', post %></button>
-            <% if user_signed_in? %>
-              <% if post.user_id == current_user.id %>
-                <button type="button" class="btn btn-default navbar-btn"> <%= link_to 'Edit', edit_post_path(post) %></button>
-                <button type="button" class="btn btn-default navbar-btn"><%= link_to 'Destroy', post, method: :delete, data: { confirm: 'Are you sure?' } %></button>
-              <% end %>
-            <% end %>
-          </div>
+                        <g transform="translate(0.000000,1419.000000) scale(0.100000,-0.100000)"
+                          fill="red" stroke="none">
+                          <path d="M2970 13410 c-560 -54 -1010 -269 -1419 -680 -398 -399 -634 -882
+                          -717 -1465 -22 -150 -24 -493 -5 -650 83 -696 363 -1310 1078 -2369 222 -329
+                          497 -711 983 -1366 1371 -1849 1954 -2697 2378 -3456 266 -478 426 -841 616
+                          -1401 52 -153 107 -305 121 -337 22 -50 28 -57 39 -45 17 18 60 131 143 374
+                          257 754 671 1536 1354 2562 430 645 779 1126 1463 2013 969 1257 1288 1717
+                          1595 2300 315 596 490 1105 568 1655 26 187 26 678 -1 850 -64 416 -216 786
+                          -452 1102 -82 108 -269 300 -389 396 -359 290 -696 426 -1245 503 -159 23
+                          -660 26 -785 6 -440 -72 -879 -311 -1269 -691 -347 -339 -585 -700 -776 -1176
+                          -88 -221 -166 -384 -186 -392 -41 -15 -138 136 -314 492 -77 154 -176 341
+                          -220 415 -458 762 -1093 1215 -1880 1341 -144 23 -523 34 -680 19z"/>
+                        </g>
+                  </svg>
+                  <%= post.likes.count %> <%= (post.likes.count) == 1 ||  (post.likes.count) == 0 ? 'Like' : 'Likes'%>
+                  <% if user_signed_in? %>
+                    <% like = current_user.likes.find_by(post: post) %>
```

Comment:
Hạn chế viết kiểu này nhé, viết query ở controller xong trong view này in ra thôi